src/components/Chip/Chip.module.css

```diff
@@ -0,0 +1,3 @@
+.Chip {
```

Comment:
Something I've noticed is that if we use the component name as the wrapper class, then css modules doubles up on it. I wonder if we should use a convention like `container` or something. That's what I did. 🤷 

![Screen Shot 2022-08-02 at 5 21 11 PM](https://user-images.githubusercontent.com/157195/182483816-68faeef0-21fd-4375-a022-7246c3398114.png)